driver/normalizer/normalizer.go

```diff
@@ -74,16 +74,22 @@ var Preprocessors = []Mapping{
 
 // Normalizers is the main block of normalization rules to convert native AST to semantic UAST.
 var Normalizers = []Mapping{
+	//FIXME(bzz): save all 3 in new parent Group \w ordered comments
+	Map(
+		Part("_", Fields{
+			{Name: "leadingComments", Drop: true, Op: Check(OfKind(nodes.KindNil | nodes.KindArray), Empty(Any()))},
```

Comment:
Also, I would propose to have 3 different mapping for each field (without `Opt`/`Drop`) - they may appear separately.
And right now this matches against all nodes since all 3 fields are optional (`Drop`).

Reply:
🤦‍♂️ Thanks! I knew but again forgot that those got treated as And!